schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaOperator.java

```diff
@@ -377,7 +376,7 @@ public void importTables(List<ImportTableSchema> tableSchemaList) throws SchemaL
         logger.info("Importing the table {} in the namespace {} succeeded.", table, namespace);
```

Comment:
Just found the period to be removed.

Reply:
Thanks, I fixed this one in https://github.com/scalar-labs/scalardb/pull/985/commits/a792fb361f80511671f53d75a3ae750e80bc90fb. 
And fixed others I had missed in https://github.com/scalar-labs/scalardb/pull/985/commits/d056723e82aa27909665ab9397bd349418c857f4 😢

Reply:
💯